react/features/chat/components/web/NewMessagesButton.tsx

```diff
@@ -0,0 +1,90 @@
+import { makeStyles } from '@material-ui/core';
+import React, { MouseEventHandler } from 'react';
+import { WithTranslation } from 'react-i18next';
+
+import { translate } from '../../../base/i18n/functions';
+import Icon from '../../../base/icons/components/Icon';
+import { IconArrowDown } from '../../../base/icons/svg/index';
+import { withPixelLineHeight } from '../../../base/styles/functions.web';
+export interface INewMessagesButtonProps extends WithTranslation {
+    inputChatHeight: number;
+    onGoToFirstUnreadMessage: MouseEventHandler;
+}
+
+const useStyles = makeStyles((theme: any) => {
+    return {
+        container: {
+            position: 'absolute',
+            left: 'calc(50% - 72px)',
+            bottom: '15px'
+        },
+
+        newMessagesButton: {
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            width: '143px',
+            height: '32px',
+            padding: '6px 8px',
+            border: 'none',
+            borderRadius: theme.shape.borderRadius,
+            backgroundColor: theme.palette.section01,
+            boxShadow: '0px 3px 16px rgba(0, 0, 0, 0.6), 0px 0px 4px 1px rgba(0, 0, 0, 0.25)',
+
+            '&:hover': {
+                backgroundColor: theme.palette.action02Hover
+            },
+
+            '&:active': {
+                backgroundColor: theme.palette.action02Active
+            }
+        },
+
+        arrowDownIconContainer: {
+            height: '20px',
+            width: '20px',
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+
+            '& svg': {
+                fill: theme.palette.uiBackground,
+                transform: 'matrix(-1, 0, 0, 1, 0, 0);'
```

Comment:
Why is this necessary?

Reply:
I have deleted, I took it from Figma, not really necessary